ddpui/ddpairbyte/airbytehelpers.py

```
@@ -285,6 +290,24 @@ def create_connection(org: Org, payload: AirbyteConnectionCreate):
+        # Upsert the AirbyteConnection block only if the user configured casts. Blocks
+        # are created lazily — connections without post_sync_transform never get a block.
+        if payload.post_sync_transform:
+            try:
+                extra = build_connection_block_extra(sync_org_task)
+                prefect_service.upsert_airbyte_connection_block(
+                    server_block_name=org_airbyte_server_block.block_name,
+                    connection_id=airbyte_conn["connectionId"],
+                    connection_name=payload.name,
+                    extra=extra,
+                )
+            except Exception as err:  # pylint: disable=broad-exception-caught
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)